client/components/SearchBar.js

```diff
@@ -5,7 +5,7 @@ import base64url from 'base64-url';
 module.exports = () =>
   (
     <div className="row">
-      <div id="formDiv" className="col-lg-6">
+      <div id="formDiv" className="col-lg-12">
```

Comment:
I changed this in my branch as well, except left it as a col-lg-6 and put buffers of col-lg-3 on either side.  It kept the shape a little better and didn't stretch it out as much, but we can look at both and see which fits with the rest of the content.